### PR TITLE
updated a c course in the list

### PR DESCRIPTION
**Pull request by thevishehgarg**
I have added one course in c its from youtube by a very good coding channel code with harry . I have personally learned c from thid course

## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo

## Checklist:
- [ ] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md)
- [ ] Search for duplicates.
- [ ] Include author(s) and platform where appropriate.
- [ ] Put lists in alphabetical order, correct spacing.
- [ ] Add needed indications (PDF, access notes, under construction)

## Followup

- Check the output of Travis-CI for linter errors!

### CODE REVIEW

File: courses/free-courses-en.md [hunk 1]
```diff
@@ -156,6 +156,7 @@
 
 ### C
 
+* [C language tutorials](https://www.youtube.com/playlist?list=PLu0W_9lII9aiXlHcLx-mDH1Qul38wD3aR) - Code with harry
```
Comment (davorpa) [inline courses/free-courses-en.md:159]: - Playlist is called: `C Language Tutorials In Hindi` and creator channel `CodeWithHarry`
- Complete platform: YouTube

```suggestion
* [C Language Tutorials In Hindi](https://www.youtube.com/playlist?list=PLu0W_9lII9aiXlHcLx-mDH1Qul38wD3aR) - 
CodeWithHarry (YouTube)
```

**Review by davorpa — CHANGES_REQUESTED** [review]
Following Contributors documentation...

We prefer the resources original titles and creators. It's also a good practice completing platform info as notes.

**Comment by davorpa** [comment]
It's seems all playlist videos are in hindi so it should be moved to that alphabetized list but there are a little problem. Checking this listing, resource is already registered:

https://github.com/EbookFoundation/free-programming-books/blob/7023637f37d453a4ccee61c49af7eaaeccc9e8af/courses/free-courses-hi.md#L23-L30

File: courses/free-courses-en.md (hunk 1 above)
Comment (eshellman) [reply courses/free-courses-en.md:159]: Youtube is an exception to the platform name guideline - don't need to add it; As @davorpa notes, please use "C Language Tutorials In Hindi" as the title

**Comment by eshellman** [comment]
duplicate (that's one of the reasons we ask for the title _from the resource_.

**Pull request closed without merging**